src/server/verification/verificationAPI.js

```diff
@@ -245,6 +245,8 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
 
       log.info('otp request:', { user, body })
 
+      const onlyCheckAlreadyVerified = body.onlyCheckAlreadyVerified
```

Comment:
onlyCheckAlreadyVerified was set as constant client side and it's always true.
if this will be the only behavior - we don't need this flag, just skip send OTP if is already verified
or you could do a new env variable for this option

Reply:
it is not always true, there are other places in the DAPP where you call this endpoint without this param

Reply:
ok